api/MediaDevices.json

```diff
@@ -208,40 +211,65 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDevices/getUserMedia",
           "support": {
             "webview_android": {
-              "version_added": null
+              "version_added": "53",
+              "notes": [
+                "Older versions of Chrome and Opera implement navigator.webkitGetUserMedia, the prefixed version of the legacy navigator.getUserMedia method. From version 47 to 52, the promise-based interface was only available through the adapter.js polyfill, or using the flag chrome://flags/#enable-experimental-web-platform-features. Starting with version 53, the promise-based interface is on by default, though that interface is still not available through navigator.",
+                "Chrome throws error if the page serving the script is loaded from insecure origin (i.e. HTTP)."
+              ]
             },
             "chrome": {
-              "version_added": null
+              "version_added": "53",
+              "notes": [
+                "Older versions of Chrome and Opera implement navigator.webkitGetUserMedia, the prefixed version of the legacy navigator.getUserMedia method. From version 47 to 52, the promise-based interface was only available through the adapter.js polyfill, or using the flag chrome://flags/#enable-experimental-web-platform-features. Starting with version 53, the promise-based interface is on by default, though that interface is still not available through navigator.",
+                "Chrome throws error if the page serving the script is loaded from insecure origin (i.e. HTTP)."
+              ]
             },
             "chrome_android": {
-              "version_added": null
+              "version_added": "53",
+              "notes": [
+                "Older versions of Chrome and Opera implement navigator.webkitGetUserMedia, the prefixed version of the legacy navigator.getUserMedia method. From version 47 to 52, the promise-based interface was only available through the adapter.js polyfill, or using the flag chrome://flags/#enable-experimental-web-platform-features. Starting with version 53, the promise-based interface is on by default, though that interface is still not available through navigator.",
```

Comment:
Like before, the story for Chrome for Android is rather complex. I think this note can be replaced by two things:

* A new support statement, `"version_added": "47"`, `"version_removed": "53"`, and a `
"flags"` entry for the "Experimental Web Platform features" preference (value: "Enabled")
* A new note: `"Older versions of Chrome implement <code>navigator.webkitGetUserMedia</code>, a prefixed form of the legacy <a href='https://developer.mozilla.org/docs/Web/API/Navigator/getUserMedia'><code>navigator.getUserMedia</code></a> API."`

Reply:
Agree